crates/tabby/src/main.rs

```diff
@@ -136,5 +136,6 @@ fn to_local_config(model: &str, parallelism: u8, device: &Device) -> ModelConfig
         parallelism,
         num_gpu_layers,
         enable_fast_attention: None,
+        context_size: 4096,
```

Comment:
is there a way to reuse default definition in `config.rs`?

Reply:
yes, but we must use it as a public function